FWCore/ParameterSet/src/ConfigurationDescriptions.cc

```diff
@@ -147,6 +147,54 @@ namespace edm {
                       std::cref(baseType_),
                       std::cref(pluginName_),
                       std::ref(usedCfiFileNames)));
+    if (defaultDescDefined_) {
+      writeClassFile(defaultDesc_);
+    } else if (descriptions_.size() == 1) {
+      writeClassFile(descriptions_.begin()->second);
+    }
```

Comment:
Does this mean if the `ConfigurationDescriptions` contains many `ParameterSetDescriptions` of which none is the default (i.e. added with `ParameterSetDescription::addDefault()`), class python file is not created?

Reply:
That is correct.